webapp/Dockerfile

```diff
@@ -1,7 +1,7 @@
 ##################################################################################
 # BASE (Is pushed to DockerHub for rebranding) ###################################
 ##################################################################################
-FROM node:20.7.0-alpine3.17 as base
+FROM node:20.2.0-alpine3.17 as base
```

Comment:
This change is not necessary.
The [last update here](https://github.com/Ocelot-Social-Community/Ocelot-Social/pull/6749) was already from `20.6.0-alpine3.17` to `20.7.0-alpine3.17`.
The Node issue we have in Kubernetes seems to be the container, which contains this repo.

Reply:
I don't get it.
What we have to change then?

The error is on Kubernetes in the webapp and this is used there. Nothing else as far I know.

Reply:
My bad. 
Ignore the comment. Your changes are correct.